src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map(film => (
+      <MovieCard movie={film} key={film.imdbId} />
+    ))}
+  </div>
```

Comment:
use semantic tags for list rendering

Reply:
ok! didn't know we can change initial layout

Reply:
Done!